source/cfn/templates/neptune.template

```diff
@@ -282,7 +276,6 @@ Resources:
   NeptuneDBInstance:
     Type: AWS::Neptune::DBInstance
     Properties:
-      AvailabilityZone: !Ref AvailabilityZone0
```

Comment:
What's the consequence of omitting this? I see you are still fetching the AZ from a static mapping in your VPC, shouldn't this be more like the following to not break existing deployments?

```
AvailabilityZone: !If
  - CreateVpc
  - !Ref AvailabilityZone0
  - !Ref AWS::NoValue
```

Reply:
Ah yes, good catch!

Reply:
*fixed